packages/codebytes/src/editor.tsx

```diff
@@ -9,6 +9,7 @@ import { CopyIcon } from '@codecademy/gamut-icons';
 import styled from '@emotion/styled';
 import React, { useState } from 'react';
 
+import { CodebytesChangeHandlerMap } from '.';
```

Comment:
You'll probably want to separate out a file like `types.ts` or similar to avoid circular references. This file imports index which imports this file...

Reply:
cc @zippyzow